src/main/java/com/onlinebookstore/repository/OrderRepository.java

```diff
@@ -0,0 +1,26 @@
+package com.onlinebookstore.repository;
+
+import com.onlinebookstore.model.Order;
+import jakarta.transaction.Transactional;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface OrderRepository extends JpaRepository<Order, Long> {
+    @Query("SELECT o FROM Order o LEFT JOIN FETCH o.orderItems LEFT JOIN FETCH o.user")
+    List<Order> findAllOrders(Pageable pageable);
+
+    @Modifying
+    @Transactional
+    @Query("UPDATE Order o SET o.status = :status WHERE o.id = :orderId")
+    void updateOrderByStatus(Long orderId, Order.Status status);
+
+    @Query("SELECT o FROM Order o LEFT JOIN FETCH o.orderItems WHERE o.id = :id")
```

Comment:
```suggestion
    @EntityGraph(attributePaths = {"orderItems", "orderItems.book"}))
```

Reply:
This works: "@EntityGraph(attributePaths = "orderItems")"